bitnami/nginx/templates/deployment.yaml

```
@@ -131,6 +131,7 @@ spec:
             - -ec
             - |
               [[ -f "/opt/bitnami/scripts/git/entrypoint.sh" ]] && source "/opt/bitnami/scripts/git/entrypoint.sh"
+              git config --global --add safe.directory /app
               while true; do
                   cd /app && git pull origin {{ .Values.cloneStaticSiteFromGit.branch }}
```

I would avoid having to use the `git config --global` command as it tries to persist the configuration in `/.gitconfig` and that file may not be writable by an unprivileged user. I see it was already mentioned in the associated issue https://github.com/bitnami/charts/issues/20682#issuecomment-1844993985, and to workaround that, you are setting `HOME=/tmp`, but that is not set by default, right?

What if we just do the following instead?

```suggestion
              while true; do
                  cd /app && git -c safe.directory=/app pull origin {{ .Values.cloneStaticSiteFromGit.branch }}
```